### PR TITLE
minor bug fixes, refactoring and style

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/Endpoint.java [hunk 1]
```diff
@@ -496,7 +496,18 @@ else if (packet instanceof RtcpSrPacket)
...
+            {
+                if (logger.isDebugEnabled())
+                {
+                    logger.debug(
+                        "dropping an sr from ssrc="
+                            + rtcpSrPacket.getSenderSsrc()
+                            + ", timestamp="
+                            + rtcpSrPacket.getSenderInfo().getRtpTimestamp());
+                }
+                return;
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Endpoint.java]: Doesn't this drop all SRs for audio?

Comment (bgrozev) [reply src/main/java/org/jitsi/videobridge/Endpoint.java]: Note that we're already filtering out the SRs for video streams that the BC doesn't accept (in `Endpoint.wants()`/`BitrateController.accept()`.

Comment (gpolitis) [reply src/main/java/org/jitsi/videobridge/Endpoint.java]: right, i forgot that, i'll revert this

File: src/main/java/org/jitsi/videobridge/cc/BitrateController.java [hunk 2]
```diff
@@ -1545,6 +1523,15 @@ int getTargetIndex()
                 ? ratedIndices[ratedTargetIdx].encoding.getIndex() : -1;
         }
 
+
+        int getPreferredIndex()
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/cc/BitrateController.java]: can this be private or protected?

Comment (gpolitis) [reply src/main/java/org/jitsi/videobridge/cc/BitrateController.java]: all these methods (gettargetindex, gettargetbitrate, getpreferredindex, etc) already are package-private and intellij idea doesn't raise a warning. If we change them to be protected then intellij idea gives a warning. We can make them private (i don't see the reason why but i don't really mind either), but i think they should all be the same for concistency

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/cc/BitrateController.java]: It's always good to scope methods as tightly as they can be. This better isolates the class, makes testing and future refactoring easier and better communicates how methods are supposed to be used (or not be used externally).

Comment (bgrozev) [reply src/main/java/org/jitsi/videobridge/cc/BitrateController.java]: Note that all these are members of a private class, so they are already only accessible from `BitrateController`

**Pull request merged**